lib/vector-config/src/lib.rs

```diff
@@ -81,6 +81,32 @@
 // can essentially define flags i.e. `docs:templateable` as a metadata value for marking a field as working with
 // Vector's template syntax, since doing `templateable = true` is weird given that we never otherwise specifically
 // disable it. In other words, we want a way to define feature flags in metadata.
+//
+// TODO: Should we add a way, and/or make it the default, that if you only supply a description of a
+// field, it concats the description of the type of the field? for example, you have:
```

Comment:
👍 does feel like a nice addition.

Reply:
Yeah, it would make it a lot easier to only partially customize the description of a field using a type that already has a rich description.